libs/utils/src/lib/gdpr-cookie/cookie-banner/cookie-banner.component.ts

```diff
@@ -18,6 +19,10 @@ import { createModalData } from '@blockframes/ui/global-modal/global-modal.compo
 export class CookieBannerComponent implements OnInit {
 
   public hasAccepted = false;
+  public form = new FormGroup({
+    behaviorCookie: new FormControl(false),
+    chatCookie: new FormControl(false),
```

Comment:
```suggestion
    hotjar: new FormControl(false),
    intercom: new FormControl(false),
```
For the users we avoid use technical terms in the front-end but in our code we can just use hotjar and intercom ;)